packages/certificate-verification-js/src/index.ts

```diff
@@ -9,13 +9,21 @@ import { Principal } from '@dfinity/principal';
 import { PipeArrayBuffer, lebDecode } from '@dfinity/candid';
 import { CertificateTimeError, CertificateVerificationError } from './error';
 
-export async function verifyCertification(
-  canisterId: Principal,
-  encodedCertificate: ArrayBuffer,
-  encodedTree: ArrayBuffer,
-  rootKey: ArrayBuffer,
-  maxCertificateTimeOffsetMs: number,
-): Promise<HashTree> {
+export interface CertificationParams {
```

Comment:
```suggestion
export interface VerifyCertificationParams {
```

If you wouldn't like to associate the name exactly to the function then you could still do `CertificationParams` but then have an alias like `type VerifyCertificationParams = CertificationParams`

Reply:
I think associating the name to the function makes sense since the interface is dedicated for it, thanks for the suggestion!